proposed/container-configuration-meta.md

```diff
@@ -97,17 +99,54 @@ In this section, we will describe the list of features that we can consider to f
 
 1. Ability to directly debug the code generating the services (using Xdebug or a similar tool). This is typically a feature available in service providers and not available in configuration files.
 
+#### 4.1.2. Vote on those features
 
-When the list of features is complete, I propose we create a poll on each feature, ranking them from -- to ++:
+A vote is currently being cast on the list of features that must be kept.
+Anyone interested is [welcome to vote](https://github.com/container-interop/fig-standards/issues/9)
+Vote results are [aggregated in the wiki](https://github.com/container-interop/fig-standards/wiki/Vote-results-for-the-important-feature-list)
 
-- -- = highly counterproductive
-- - = not needed
-- / = indifferent
-- + = nice to have
-- ++ = absolutely needed
+### 4.2. Format
 
+The following formats are considered for implementing the configuration:
 
-### 4.2. The format
+- a standard static format, for example based on XML, JSON, YAML... This has been discussed by `container-interop` members (with a preference for XML), but was not tested.
+- standard PHP objects/interfaces representing container definitions. This has been tested in [container-interop/definition-interop](https://github.com/container-interop/definition-interop)
+- standard service providers. This has been tested in [container-interop/service-provider](https://github.com/container-interop/service-provider)
 
-TODO
+#### 4.2.1 Feature support
+
+List of features that can be supported for each configuration format.
+
+Note: only features with an average score of "+" or "++" have been kept.
```

Comment:
OK, I think we should have all features in there, at least until we have a more complete poll. And I understand the rating now, but maybe it could be simpler and more explicit? Like "yes", "no" and "complicated"? I don't think we need many sub-levels.